pkg/bmc/virtualmedia_validation.go

```diff
@@ -0,0 +1,57 @@
+package bmc
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stmcginnis/gofish"
+	"github.com/stmcginnis/gofish/redfish"
+)
+
+// ValidateVirtualMedia returns true, nil when the
+// ValidateVirtualMedia tests the connection information for
+// the host to verify if has support for Virtual Media.
+func ValidateVirtualMedia(address string, user string, password string, insecure bool) (bool, error) {
+	bmcConfig := gofish.ClientConfig{
+		Endpoint: address,
+		Username: user,
+		Password: password,
+		Insecure: insecure,
+	}
+
+	client, err := gofish.Connect(bmcConfig)
+	if err != nil {
+		return false, errors.Wrap(err, "Could not connect to redfish BMC.")
```

Comment:
errors should begin with lower case, and not end with punctuation

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Done